Makefile

```diff
@@ -11,8 +11,7 @@ pylint:
 	pylint nameko -E
 
 pytest:
-	coverage run --concurrency=eventlet --source nameko -m pytest test
-	coverage report --fail-under=100
+	py.test test --cov
```

Comment:
Yes, that's why the travis build is broken!

Alternatively we could land https://github.com/onefinestay/nameko/pull/265